### ISSUE TITLE
showIntroMessage doesn't work

### ISSUE THREAD

**Issue by ink-spot** [Instabug/Instabug-iOS#112]
The `showIntroMessage` doesn't seem to work anymore since the 6.4.1 update.
I set the invocation event to shake (same result with other events), and when calling the `showIntroMessage`, nothing happen.

```
[Instabug startWithToken:token invocationEvent:IBGInvocationEventShake];
...
[Instabug showIntroMessage];
```

This worked well with the 6.3 version.

**Comment by Figo0o** [comment]
Hey, 

Can you please make sure that you haven't called `[Instabug setIntroMessageEnabled:NO];` in your code. 

One more thing, if Instabug is presented the message won't appear. 

If that doesn't work can you share Instabug code used ? so we can reproduce it. 

Thanks!

**Comment by ink-spot** [comment]
Hey,
good catch, I was actually calling `[Instabug setIntroMessageEnabled:NO];` earlier to prevent the info message to pop at a random time (I need to be able to show it manually).
With 6.4.1, if I don't disable the intro message, will it pop randomly once like in the previous version?

**Comment by Figo0o** [comment]
This should be fixed in the next release. I can send a custom build for you if this issue is blocking you. 

Please Let me know. 

Thanks,

**Comment by ink-spot** [comment]
No don't worry, for now I just enable the intro message right before showing it, and disable it again right after, and everything seems to work fine.
I'll stick with that solution for now, and I'll check the next release when it'll be live.
Thanks a lot for your answers!

**Comment by crododile** [comment]
@Figo0o was this issue resolved in 7.0? I do not see anything about it in the  release notes and am having difficulties with showIntroMessage ( it doesn't do anything, even after re-enabling as described here )

**Comment by Figo0o** [comment]
Hello @crododile, Yes this should be solved. Can you please provide me with invocation event used ?
